### ISSUE TITLE
RFP - pikchr

### ISSUE THREAD

**Issue by manipuladordedados** [chocolatey-community/chocolatey-package-requests#1512]
## Checklist

- [x] The package I am requesting does not already exist on https://community.chocolatey.org/packages;
- [x] There is no open issue for this package;
- [x] The issue title starts with 'RFP - ';
- [x] The download URL is public and not locked behind a paywall / login;

## Package Details

Software project URL : https://pikchr.org
Direct download URL for the software / installer:  https://github.com/manipuladordedados/pikchr/releases/download/v0.0.0-1/pikchr.exe
Software summary / short description: Pikchr is a PIC-like markup language for diagrams in technical documentation.

**Comment by pauby** [comment]
The download file you've linked to is a C source file. Is there no installer available?

**Comment by pauby** [comment]
How would a user be able to determine that what you have compiled is the official source without any additions, malicious or otherwise?

**Comment by manipuladordedados** [comment]
> How would a user be able to determine that what you have compiled is the official source without any additions, malicious or otherwise?
> 
> Nor that I'm not making any suggestions as to your intent.

Even though I'm the one generating the binary, you can clone my repository. if you look at the CI script, you'll see that it downloads from the official repository and uses Chocolatey to install dependencies. 
It's easy to audit.

**Comment by pauby** [comment]
The Chocolatey Community Repository will only accept packages that download or embed binaries from official sources.

**Comment by manipuladordedados** [comment]
Is there an official Chocolatey repository on GitHub to which I can submit the CI script and thus use its repositories instead of mine? If that's not possible, I don't know of any alternative to contribute to this package, and perhaps it would be better to close the issue, as the official developers of Pikchr do not provides binaries for Windows.

**Comment by pauby** [comment]
That repository would also not be official so it's going to be the same situation.

We've come across this a few times and I'm fairly sure it always ends up the same.

@AdmiringWorm @TheCakeIsNaOH any thoughts?

**Comment by AdmiringWorm** [comment]
Yeah, as long as it is not officially provided by the software authors, or at a minimum linked to from the software author's website (thus making it semi-official) it can't be used.

I think the only way to provide this until there are official binaries is to compile the software as part of the package installation (I remember seeing one or two packages that did it a few years ago, but I don't remember which ones).

**Comment by manipuladordedados** [comment]
> I think the only way to provide this until there are official binaries is to compile the software as part of the package installation (I remember seeing one or two packages that did it a few years ago, but I don't remember which ones).

Could you tell me which ones they are so I can use them as inspiration and do the same?

**Comment by pauby** [comment]
I'm going to go ahead and close this as there has been no update in almost a year. We can always reopen again if necessary.
